### PR TITLE
Use asciidoc as the manual page source format

### PR DESCRIPTION
**Pull request by dkopecek**
To generate the manual page, you can use the a2x utility
from the asciidoc package:

 $ a2x -v -f manpage jose.1.adoc
 ...

### CODE REVIEW

File: doc/man/jose-fmt.1.adoc [hunk 1]
```diff
@@ -1,163 +1,168 @@
...
-==========================================================
+jose-fmt(1)
+===========
+:doctype: manpage
 
-## SYNOPSIS
+== NAME
 
-`jose fmt` [OPTIONS]
+jose-fmt - Converts JSON between serialization formats
```
Comment (npmccallum) [inline doc/man/jose-fmt.1.adoc]: Why do the command names have hyphens? Those aren't the commands.

Comment (dkopecek) [reply doc/man/jose-fmt.1.adoc]: I'll fix the hyphens, it's caused by the script I've created for the conversion. Didn't catch that during my review.

**Comment by npmccallum** [comment]
@dkopecek Can you update this? I am hoping to merge this tomorrow.

**Comment by simo5** [comment]
@dkopecek any chance we can update so we can merge this ?

**Comment by dkopecek** [comment]
> @dkopecek any chance we can update so we can merge this ?

Hi, I'm no longer active in this project. Anyway, feel free to take ownership of the patch and finish whatever is needed!

**Comment by sergio-correia** [comment]
> > @dkopecek any chance we can update so we can merge this ?
> 
> Hi, I'm no longer active in this project. Anyway, feel free to take ownership of the patch and finish whatever is needed!

@dkopecek @simo5: I will take over this PR and update/resubmit it in the next days.

**Comment by dkopecek** [comment]
@sergio-correia Ok, thanks! Closing this one.

**Pull request closed without merging**